node/node.go

```diff
@@ -105,6 +105,13 @@ type Config struct {
 	// for connections to the remote API (including the raft service).
 	AdvertiseRemoteAPI string
 
+	// DefaultAddrPool specifies default subnet pool for global scope networks
+	DefaultAddrPool []*net.IPNet
```

Comment:
Silly question, but if multiple pools have been configured (e.g. a `172.xx` pool and a `192.xx pool`), will all of those pools be used? (i.e., networks use `172.xx` until no more addresses are available, then the next network will use `192.xx` ?), or will the daemon pick the first pool that doesn't overlap with a physical network, and only use that pool from then on?

If only one pool is selected (and used from then on), we may need to expose _which_ pool is picked somewhere

Reply:
if we have 20.20.0.0/16  and 30.30.0.0/16 pool, First it will  pickup from 20.20.0.1/24 , 20.20.1.1/24 , etc. Then it will come back to 30.30.0.0/16. This works same even with our current inbuilt default pool.  
We can add it as separate P1 in our PRD/design document and work on it.

Reply:
@thaJeztah yes this is actually how local scope networks work today where we have two default ranges (172 and 192). We may plan for specific pool selection in the future but right now it is just one continuous range. In a nutshell this allows a customer to string together a larger address range when their internal available pools are already scattered.